source/renderer/app/components/wallet/transactions/Transaction.js

```diff
@@ -415,9 +420,36 @@ export default class Transaction extends Component<Props, State> {
             >
               <div>
                 <h2>{intl.formatMessage(messages.fromAddresses)}</h2>
-
                 {fromAddresses(data.addresses.from, data.id)}
 
+                {data.addresses.withdrawals.length ? (
+                  <>
+                    <h2>{intl.formatMessage(messages.fromRewards)}</h2>
+                    {data.addresses.withdrawals.map((address, addressIndex) => (
+                      <div
+                        // eslint-disable-next-line react/no-array-index-key
```

Comment:
Wouldn't `address` be enough, here, as it's just an internal react thing?

Reply:
No @daniloprates :(
We had cases where there were address duplicates and then adding the index fixes the React key issues!